packages/events-example/src/commands/reconcile/index.ts

```diff
@@ -1,10 +1,9 @@
 import { Command } from '@oclif/core';
 import Nile, { Instance, NileApi } from '@theniledev/js';
 
-import { ReconciliationPlan } from '../../model/ReconciliationPlan';
-
-import { pulumiS3, PulumiAwsDeployment } from './lib/pulumi';
-import { flagDefaults } from './flagDefaults';
+import { pulumiS3, PulumiAwsDeployment } from '../../pulumi';
```

Comment:
I think this makes sense under a sub directory. as we continue to add new integrations, it will clog up the base path of the repo. `commands` and `pulumi` are not on the same level of functionality.

Reply:
This all gets ripped out into different packages in the `nile-agent` lib refactor I've got going. This stuff will end up in `/packages/pulumi`.